src/main/java/core/basesyntax/model/Report.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Report {
+    public static final Map<String, Integer> REPORT_TEMPLATE = new HashMap<>();
```

Comment:
What is the `REPORT_TEMPLATE` map? By name, I don't understand what contains this variable

Reply:
Renamed.